### PR TITLE
OGM-1433 Support drop collection as native query for MongoDB

### PR DESCRIPTION
**Pull request by DavideD**
https://hibernate.atlassian.net/browse/OGM-1433

Follows: https://github.com/hibernate/hibernate-ogm/pull/998

### CODE REVIEW

**Comment by DavideD** [comment]
@fax4ever can you have quick look at this PR and merge if you don't find anything too weird? Thanks

**Comment by Sanne** [comment]
s/colleciton/collection/

File: mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/query/parsing/nativequery/impl/NativeQueryParser.java [hunk 1]
```diff
@@ -367,6 +369,16 @@ public Rule MapReduce() {
 		);
 	}
 
+	public Rule Drop() {
+		return Sequence(
+				"drop",
```
Comment (fax4ever) [inline mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/query/parsing/nativequery/impl/NativeQueryParser.java]: s/"drop"/"drop "

Comment (DavideD) [reply mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/query/parsing/nativequery/impl/NativeQueryParser.java]: Fixed.

I wonder why we need the space though. Anyway, I've updated the code

**Comment by DavideD** [comment]
Merged https://github.com/hibernate/hibernate-ogm/pull/1002

**Pull request merged**